src/ucp/tag/eager_snd.c

```diff
@@ -199,7 +199,13 @@ void ucp_tag_eager_sync_completion(ucp_request_t *req, uint32_t flag,
 
     ucs_assertv(!(req->flags & flag), "req->flags=%d flag=%d", req->flags, flag);
     req->flags |= flag;
+
+    if (flag & UCP_REQUEST_FLAG_LOCAL_COMPLETED) {
+        ucp_request_send_track(req);
+    }
```

Comment:
can we track before we started sending?
what if we get the remote reply before local send completion?

Reply:
done